app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaApp.kt

```diff
@@ -104,6 +105,19 @@ fun NiaApp(appState: NiaAppState, modifier: Modifier = Modifier) {
                     )
                 }
             }
+            LaunchedEffect(errorMessage) {
+                errorMessage?.let {
+                    val snackBarResult = snackbarHostState.showSnackbar(
+                        message = it,
+                        actionLabel = "Continue",
+                        duration = Indefinite,
```

Comment:
Is there any reason the duration should be `Indefinite`?

Reply:
Using `Indefinite` keeps the notification on the window until the user clicks the action button, which will indicate the user saw each individual error message; this is similar to the implementation of the `offline` snacker notification. I believe this could be implemented to remove the error automatically with shorter duration, similar to the way `Bookmarks` removed in the` BookmarksScreen`, if desired.

Reply:
Now, the implementation allows for selecting durations. Marking this as resolved, thank you.